### DISCUSSION TITLE
Getting torque applied by enableAngularMotor

### DISCUSSION THREAD

**Discussion by Gregwar** [bulletphysics/bullet3#3513] [General]
Hello,
When calling `enableAngularMotor` on an hinge joint, there is no way to get the impulse that is effectively applied on the joint. `getAppliedImpulse` returns an impulse that was used for both the motor and the hinge joint
How this could be achieved?
Also, what is the policy of `enableAngularMotor` to reach the given speed?

**Comment by Gregwar** [comment]
Note that `getJointFeedback()` returns NULL even when calling `enableFeedback(true)`

**Comment by Gregwar** [comment]
I eventually tried:

``` c++
btJointFeedback *fb = new btJointFeedback();
hinge->setJointFeedback(fb);
```

And then later have a look at the data populated in the joint feedback
However, there is still applied torque even if I set the maximum impulse to 0, so this is not really the information I want, any idea?

**Comment by benelot** [comment]
Can you close this issue and post your question again on the bullet physics forum? We are ready to help you there, here you only get limited attention sometimes!
